_posts/2018-08-21-OSS-by-Default-Docs.markdown

```diff
@@ -0,0 +1,153 @@
+- By making potential private we could have one, authoritative source of truth for Artsy engineering newcomers
+
+These are all great, reasonable arguments and so we made the repo private - but with hindsight, closing the repo
+split contributors to our documentation. For example, the mobile team split moved their documentation into separate
+repo the week potential was made private. The web team used the blog, or kept notes in a per-project basis.
+Culturally, our main documentation repo was considered the domain of our platform team. The platform team had the
+most cross-cutting concerns, and were also the team with the strongest need for documenting private information like
+domain models, outages, system failures, server access details and contact details.
+
+By 2016, we had successfully [de-silo'd][desilo] mobile engineering at Artsy via React Native, so the mobile team
+wasn't a centralized team with resources anymore. This meant that new docs shouldn't really live inside that the
+artsy/mobile repo. The front-end teams had been using the blog posts and public gists to keep track of
+documentation, which isn't really a good pattern. GitHub gists get lost, and blog posts aren't living documents.
+
+This eventually caused enough dissonance that the front-end folk called it quits and started a new docs repo. Our
+discussion on what we want a GraphQL schema to look like definitely didn't fit in the [`artsy/mobile`][a_m] repo and
+we wanted to share it with the other GraphQL folk we were talking to, so having it in a private repo didn't make
+sense. We couldn't do editorial review against a gist, and we eventually just started a new documentation repo:
+[`artsy/guides`][a_g].
+
+Once we had a space, then new docs started coming. We documented the RFC process, and how to run retrospectives in
+the guides repo. As a guide on _how-to-do-x_ - these all made sense. What didn't make sense was that we were
+regularly repeating ourselves when talking about Artsy Engineering to the public.
+
+There wasn't a good space for that in mobile nor guides, and so a new repo was created: [`artsy/meta`][a_mt]. \
+
+Soon, this became the home of docs from anyone that preferred writing in the public. [`artsy/meta`'s][a_mt] domain
+was vague enough that anyone could document any internal processes as being something externally facing.
+
+## Open Docs by Default
+
+This came to an inflection point when I joined the platform team, and felt the need to write cross-team
+documentation that really didn't fit with of our existing domains for documentation. I believe in [leveraging my
+impact][lev], so any time writing docs should be industry grade-stuff, not only available to those lucky enough to
+be in [our GitHub org][jobs].
+
+So I spent some time debating the merits of our current infrastructure for docs:
+
+```diff
++ New folk know to start at artsy/potential
++ The platform team have a safe space for writing any private details about architecture and security
```

Comment:
"safe space" is a bit of a loaded term, politically. I think something like "private space for writing confidential details" might make more sense.